### ISSUE TITLE
Create Integration

### ISSUE THREAD

**Issue by ChiefArug** [ChiefArug/ThermalSysteams#3]
A block that consumes steam to power a steam engine (from Create)
Possibly a block (steam valve?) that can be placed on a Create steam boiler to turn steam from that into Systeams steam for powering a steam dynamo.

**Comment by SimonMeskens** [comment]
I just came to this Github looking to propose this enhancement. Turning Create steam into Systeams steam would be amazing.

**Comment by ChiefArug** [comment]
This looks like it will be more difficult that I initially thought due to Create hardcoding the boilers engine checks...

**Comment by SimonMeskens** [comment]
Yeah, I had a look at it as well, and it's a bit annoying to set up. It did seem possible, but there is no standard Forge fluid container or anything like that, which is unfortunate.

**Comment by ChiefArug** [comment]
It doesn't really have anything to do with Forge's fluid system because the boiler doesn't actually store any steam, it just calculates a stress capacity amount based on the amount of heat and water it is receiving, then the engines use that to output their rotational power.

**Comment by ChiefArug** [comment]
Looks like this would require mixining into Create, which I am not comfortable doing.
If someone else wants to PR this that would be great, however I will probably be stricter than normal in reviewing that.
